doc/source/lowlevelapi/TemplateObjects.rst

```diff
@@ -824,6 +828,50 @@ None
 Set the guid of the linked report. It takes as input a valid guid. If
 you want to set to link to no report, set the input to None
 
+.. _userdefinedREST:
+
+userdefinedREST object
+^^^^^^^^^^^^^^^^^^^^^^
+
+Inherits from TemplateREST, LayoutREST
+
+Class that corresponds to the `Userdefined`_ Layout
+template type. This template inserts a named div into the HTML output.
+This div can be filled with any HTML the user would like.  By default,
+the HTML content will be filled in with the ``TemplateREST.set_html()``
+content (if any).  This content may be placed before or after any
+child item/layout content.
+
+Specific properties include:
+
+**template.interactive_only: int**
+
+If this property is set to a non-zero value (the default), then the
+div will not be included when ADR is outputting PDF, PowerPoint or
+Offline HTML.  If set to zero, it will be included in PDF and Offline
+HTML output.
+
+**template.before_children: int**
+
+If this property is set to zero (the default), then any child content
+is placed in the report before the div.  If set non-zero, the div

```

Comment:
"the div will be placed in the report before the child content"

instead of 

"the div content will be placed in the report before the div"